### ISSUE TITLE
[Webinar] Video and Time stamps

### ISSUE THREAD

**Issue by Rajesh-GSPANN** [aemsites/danaher-abcam#179]
![image](https://github.com/user-attachments/assets/1c19be54-2152-4a70-8603-03b54d731d1e)

For Youtube or Vimeo

**Comment by Meenu-GSPANN** [comment]
Dave confirmed : Time Stamps/Chapter can be an enhancement in a future release if we can’t fit it into November.

**Comment by nagaraju-gspann** [comment]
I have worked and Completed with Webinars Video task.
As I connected with Sunny, he suggested to modify the width of the Video for Bigger Screens.
So, I need to work on the below points for the Webinars Video:
- Set Video Aspect Ratio.
- Padding top and bottom required
- Padding left and right are irrespective of figma. it may changes according to the screens.
- Video width should not go Beyond the container Layout width.
